test/new-e2e/tests/npm/config/dockercompose_httpbin.yaml

```diff
@@ -4,8 +4,8 @@ services:
     pid: host
     privileged: true
     ports:
-    - 80:8080/tcp
-    image: mccutchen/go-httpbin
+    - 80:80/tcp
+    image: public.ecr.aws/r3m4q3r9/pub-mirror-httpbin
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As per the discussion, you can add a TODO to move this to github container registry in `test-infra`

Reply:
I will do a followup PR